minitest-rails-capybara.gemspec

```diff
@@ -23,22 +23,22 @@ Gem::Specification.new do |s|
     s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<minitest-rails>.freeze, ["~> 3.0"])
-      s.add_runtime_dependency(%q<capybara>.freeze, ["~> 2.7"])
+      s.add_runtime_dependency(%q<minitest-rails>.freeze, [">= 3.0", "<= 5.2"])
```

Comment:
This should remain `~> 3.0`.

Reply:
Why is that? This gem continues to work with more recent `minitest-rails` versons.

Reply:
minitest-rails-capybara 3.x is intended to work with minitest-rails 3.x. (Both minitest-rails 3.x and minitest-rails 5.0.x support Rails 5.x). If you want a version for minitest-rails 5.0.x then we should release minitest-rails-capybara 5.0.x, but I am hesitant to do that.

I discuss my thoughts about versioning in this [blog post](https://blowmage.com/2019/06/18/minitest-rails-releases).

Reply:
Ping.

Reply:
@blowmage updated!